configure.ac

```diff
@@ -22,6 +22,22 @@ PKG_CHECK_MODULES([XEXT], [xext])
 PKG_CHECK_MODULES([XFIXES], [xfixes])
 PKG_CHECK_MODULES([IMLIB2], [imlib2])
 
+AC_ARG_WITH([libbsd],
+    AS_HELP_STRING([--without-libbsd], [Error when BSD functions are not found]))
+AC_CHECK_FUNCS([strlcpy strlcat err errx],, [LIBBSD_NEEDED=yes])
+AS_IF([test "x$LIBBSD_NEEDED" = "xyes"], [
+    AS_IF([test "x$with_libbsd" = "xno"], [
+        AC_MSG_ERROR([BSD functions not found and --without-libbsd was used])
+    ])
+    PKG_CHECK_MODULES([LIBBSD], [libbsd-overlay],,
+        AC_MSG_ERROR([BSD functions not found, libbsd is required]))
```

Comment:
This macro has a syntax error.
On systems where `libbsd` is not installed this macro generates an error and creates a junk file.
```bash
checking for strlcat... no
checking for err... yes
checking for errx... yes
checking for LIBBSD... no
configure: error: BSD functions not found
elif test yes = untried; then
        { printf %s\n configure:5389:
```
This error creates a new file called: `no >&5?printf %sn no >&6; }??as_fn_error 0 BSD`

```bash
 stat no\ \>\&5$'\n'printf\ %sn\ no\ \>\&6\;\ \}$'\n'$'\t'as_fn_error\ 0\ BSD
  File: 'no >&5'$'\n''printf %sn no >&6; }'$'\n\t''as_fn_error 0 BSD'
  Size: 116             Blocks: 24         IO Block: 4096   regular file
```
 
 Patch:
 ```patch
 
diff --git a/configure.ac b/configure.ac
index 769b8b7..563f5f8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -30,7 +30,7 @@ AS_IF([test "x$LIBBSD_NEEDED" = "xyes"], [
         AC_MSG_ERROR([BSD functions not found and --without-libbsd was used])
     ])
     PKG_CHECK_MODULES([LIBBSD], [libbsd-overlay],,
-        AC_MSG_ERROR([BSD functions not found, libbsd is required]))
+        [AC_MSG_ERROR([BSD functions not found, libbsd is required])])
 ])
```